ste/mgr-JobPartTransferMgr.go

```diff
@@ -748,7 +761,7 @@ func (jptm *jobPartTransferMgr) failActiveTransfer(typ transferErrorCode, descri
 			// display a clear message
 			common.GetLifecycleMgr().Info(fmt.Sprintf("Authentication failed, it is either not correct, or expired, or does not have the correct permission %s", err.Error()))
 			// and use the normal cancelling mechanism so that we can exit in a clean and controlled way
-			jptm.jobPartMgr.(*jobPartMgr).jobMgr.CancelPauseJobOrder(common.EJobStatus.Cancelling())
+			jptm.jobPartMgr.(*jobPartMgr).jobMgr.CancelPauseJobOrder(common.EJobStatus.Failed())
```

Comment:
did u test it thoroughly and make sure the job status gets set correctly

Reply:
I tried setting failed and we are getting Failed status for auth issue.

Reply:
Ok, thanks @nitin-deamon